dashboards-observability/public/components/visualizations/charts/lines/line.tsx

```diff
@@ -46,9 +43,7 @@ export const Line = ({ visualizations, layout, config }: any) => {
   }: IVisualizationContainerProps = visualizations;
   const { dataConfig = {}, layoutConfig = {}, availabilityConfig = {} } = userConfigs;
 
-  const yaxis = dataConfig[AGGREGATIONS]
-    ? dataConfig[AGGREGATIONS].filter((item) => item.label)
-    : [];
+  const yaxis = dataConfig[AGGREGATIONS] ? dataConfig[AGGREGATIONS] : [];
```

Comment:
Questions: 
- why is the filter removed
- i think this line can be further simplified as
`const yaxis = dataConfig[AGGREGATIONS] ?? [];`

Reply:
@mengweieric 
-  we can have count as aggregation without selecting any field. 
- will consider this point in next PRs